packages/dds/matrix/src/test/matrix.stress.spec.ts

```diff
@@ -269,7 +276,7 @@ describe("Matrix", () => {
             { numClients: 2, numOps: 200, syncProbability: 0.3, disconnectProbability: 0, seed: 0x84d43a0a },
             { numClients: 3, numOps: 200, syncProbability: 0.1, disconnectProbability: 0, seed: 0x655c763b },
             { numClients: 5, numOps: 200, syncProbability: 0.0, disconnectProbability: 0, seed: 0x2f98736d },
-            { numClients: 2, numOps: 200, syncProbability: 0.3, disconnectProbability: 1, seed: 0x84d43a0a },
+            { numClients: 2, numOps: 200, syncProbability: 0.2, disconnectProbability: 0.4, seed: 0x84d43a0a },
```

Comment:
Adjusted the mix a bit.  Previously, this stress config would force reconnection after every op, but it's more interesting to have a handful of ops pending during resubmission.

(There's also a pre-existing "long haul" stress that hits reconnection with hundreds of ops pending, but I only run that on my box.)